### PR TITLE
gfsv16.3 ecflow merge from parallel

### PR DESCRIPTION
**Pull request by lgannoaa**
New ecflow definition file
ecflow job script with memory requirement change from implementation parallel
Remove ecflow workflow manager package

NOTE:
This PR should be merged after #975

### CODE REVIEW

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf [hunk 1]
```diff
@@ -25,10 +25,10 @@ module load PrgEnv-intel/${PrgEnv_intel_ver}
 module load craype/${craype_ver}
 module load intel/${intel_ver}
 module load cray-pals/${cray_pals_ver}
+module load cfp/${cfp_ver}
 module load libjpeg/${libjpeg_ver}
 module load grib_util/${grib_util_ver}
-module load wgrib2/${wgrib2_ver}
-module load cfp/${cfp_ver}
+module load wgrib2/${wgrib2_veri}
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]:
```suggestion
module load wgrib2/${wgrib2_ver}
```
This has to be a fat-fingered typo.

File: ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:06:00
-#PBS -l select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=24GB
+#PBS -l select=1:mpiprocs=128:ompthreads=1:ncpus=128:mem=28GB
```
Comment (aerorahul) [inline ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf]: Please revert this change or justify why 128 cores are being requested.
The memory increase is acceptable.
@KateFriedman-NOAA will need to update `config.resources.*`

File: ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf [hunk 3]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:08:00
-#PBS -l select=1:ncpus=1:mem=2GB
+#PBS -l select=1:mpiprocs=80:ompthreads=1:ncpus=80:mem=4GB
```
Comment (aerorahul) [inline ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf]: Please revert the cores.  Memory increase is ok.

File: ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf (hunk 2 above)
Comment (aerorahul) [inline ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf]:
```suggestion
#PBS -l select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=28GB
```

File: ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf (hunk 3 above)
Comment (aerorahul) [inline ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf]:
```suggestion
#PBS -l select=1:ncpus=1:mem=4GB
```

File: ecf/scripts/gfs/atmos/obsproc/prep/jgfs_atmos_emcsfc_sfc_prep.ecf [hunk 4]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:07:00
-#PBS -l select=1:ncpus=1:mem=2GB
+#PBS -l select=1:mpiprocs=80:ompthreads=1:ncpus=80:mem=80GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/obsproc/prep/jgfs_atmos_emcsfc_sfc_prep.ecf]:
```suggestion
#PBS -l select=1:ncpus=1:mem=4GB
```

Comment (aerorahul) [reply ecf/scripts/gfs/atmos/obsproc/prep/jgfs_atmos_emcsfc_sfc_prep.ecf]: reverting core requests.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_blending_0p25.ecf [hunk 5]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=1:ompthreads=1:ncpus=1:mem=15GB
+#PBS -l select=1:ncpus=1:mem=15GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_blending_0p25.ecf]: Please undo this change to accept the developer provided resource request.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf [hunk 6]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=18:ompthreads=1:ncpus=18:mem=80GB
+#PBS -l select=1:ncpus=18:mem=80GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: Please use developer provided resource request.  Undo this change.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf [hunk 7]
```diff
@@ -25,10 +25,10 @@ module load PrgEnv-intel/${PrgEnv_intel_ver}
 module load craype/${craype_ver}
 module load intel/${intel_ver}
 module load cray-pals/${cray_pals_ver}
+module load cfp/${cfp_ver}
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: Unnecessary.  Please undo.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf (hunk 1 above)
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: Unnecessary. Please undo.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf [hunk 8]
```diff
@@ -37,8 +37,8 @@ module list
 #############################################################
 export cyc=%CYC%
 export cycle=t%CYC%z
+export USE_CFP=YES
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf:40]: accept this change

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf [hunk 9]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=11:ompthreads=1:ncpus=11:mem=80GB
+#PBS -l select=1:ncpus=11:mem=80GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf]: Please undo and accept developer provided resources.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf [hunk 10]
```diff
@@ -25,10 +25,10 @@ module load PrgEnv-intel/${PrgEnv_intel_ver}
 module load craype/${craype_ver}
 module load intel/${intel_ver}
 module load cray-pals/${cray_pals_ver}
+module load cfp/${cfp_ver}
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf]: Please undo.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf [hunk 11]
```diff
@@ -37,8 +37,8 @@ module list
 #############################################################
 export cyc=%CYC%
 export cycle=t%CYC%z
+export USE_CFP=YES
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf:40]: Accept.

File: ecf/scripts/gfs/atmos/post_processing/grib_wafs/jgfs_atmos_wafs_master.ecf [hunk 12]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:05:00
-#PBS -l select=1:mpiprocs=1:ompthreads=1:ncpus=1:mem=5GB
+#PBS -l select=1:ncpus=1:mem=2GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/atmos/post_processing/grib_wafs/jgfs_atmos_wafs_master.ecf]: Please undo and accept developer resources.

File: ecf/scripts/gfs/wave/post/jgfs_wave_prdgen_gridded.ecf [hunk 13]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=02:00:00
-#PBS -l select=1:ncpus=1:mem=1GB
+#PBS -l select=1:ncpus=1:mem=2GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/wave/post/jgfs_wave_prdgen_gridded.ecf:7]: accept

File: ecf/scripts/gfs/wave/prep/jgfs_wave_prep.ecf [hunk 14]
```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
-#PBS -l select=1:mpiprocs=65:ompthreads=1:ncpus=65:mem=200GB
+#PBS -l select=1:mpiprocs=65:ompthreads=1:ncpus=65:mem=220GB
```
Comment (aerorahul) [inline ecf/scripts/gfs/wave/prep/jgfs_wave_prep.ecf:7]: accept

File: ecf/versions/gfs.ver [hunk 15]
```diff
@@ -1,17 +1,10 @@
-#### Test only remove before code delivery
-export COMPATH=/lfs/h1/ops/canned/com/ukmet:/lfs/h1/ops/canned/com/gfs:/lfs/h1/ops/canned/com/ecmwf:/lfs/h1/ops/canned/com/nam:/lfs/h1/ops/canned/com/hourly
-####
-
+export obsproc_ver="1.0.2"
```
Comment (aerorahul) [inline ecf/versions/gfs.ver:1]: Accept this file.

Note for @KateFriedman-NOAA.
This and obsproc.ver needs to be discussed with Steven Earle about the duplication with `versions/run.ver`

File: ecf/versions/obsproc.ver [hunk 16]
```diff
@@ -1,5 +1,5 @@
 export gfs_ver="v16.2"
-export obsproc_ver="1.0.0"
+export obsproc_ver="1.0.2"
```
Comment (aerorahul) [inline ecf/versions/obsproc.ver:2]: Accept this file

**Review by aerorahul — COMMENTED** [review]
See inline comments.

File: ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf (hunk 2 above)
Comment (lgannoaa) [reply ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf]: Jan 21, 2022 a merge 45dad23d728226391e8be752f424097acc84a67a
 has changed it from:

PBS -l place=vscatter,select=1:mpiprocs=128:ompthreads=1:ncpus=128
PBS -l place=excl
to
PBS -l place=vscatter,select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=24GB
Therefore, it is save to revert it back.

File: ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf (hunk 3 above)
Comment (lgannoaa) [reply ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf]: Reverted

Comment (lgannoaa) [reply ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf]: Reverted

File: ecf/scripts/gfs/atmos/obsproc/prep/jgfs_atmos_emcsfc_sfc_prep.ecf (hunk 4 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/obsproc/prep/jgfs_atmos_emcsfc_sfc_prep.ecf]: Reverted

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf (hunk 6 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: Change undo is done.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf (hunk 7 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: undo is done

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf (hunk 1 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf]: Typo is fixed

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf (hunk 9 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf]: undo is done

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf (hunk 10 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf]: undo is done

File: ecf/scripts/gfs/atmos/post_processing/grib_wafs/jgfs_atmos_wafs_master.ecf (hunk 12 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib_wafs/jgfs_atmos_wafs_master.ecf]: undo is done

File: ecf/versions/gfs.ver (hunk 15 above)
Comment (lgannoaa) [reply ecf/versions/gfs.ver:1]: This file is NOT been used in my ecflow parallel. I use $HOMEgfs/versions/run.ver in ecflow include file.

File: ecf/versions/obsproc.ver (hunk 16 above)
Comment (lgannoaa) [reply ecf/versions/obsproc.ver:2]: This file is NOT been used in my ecflow parallel. I use $HOMEgfs/versions/run.ver in ecflow include file.

File: ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_blending_0p25.ecf (hunk 5 above)
Comment (lgannoaa) [reply ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_blending_0p25.ecf]: undo is done

File: ecf/versions/gfs.ver (hunk 15 above)
Comment (aerorahul) [reply ecf/versions/gfs.ver:1]: Ok. Great.  Thanks for the clarification.

File: ecf/versions/obsproc.ver (hunk 16 above)
Comment (aerorahul) [reply ecf/versions/obsproc.ver:2]: Understood.

**Review by aerorahul — APPROVED** [review]
Looks good.
@KateFriedman-NOAA Please take a note of the increase in the memory requests in the `.ecf` and update the `config.resources.*` accordingly.

**Review by KateFriedman-NOAA — APPROVED** [review]
Changes look good, thanks @lgannoaa !
